src/dmd/dtemplate.d

```diff
@@ -2325,8 +2325,10 @@ extern (C++) final class TypeDeduced : Type
  *      tiargs          initial list of template arguments
  *      tthis           if !NULL, the 'this' pointer argument
  *      fargs           arguments to function
+ *      failedIndex     address to store argument index of first type mismatch
```

Comment:
note that `failedIndex` can be `null`

Reply:
I've noted this for `pMessage` (which replaced `failedIndex`).

Reply:
This should be in Ddoc style:
- `Params:` 
- use `=` - then Ddoc can would throw an error about the non-existing `failedIndex`